lib/connectors/kv-memory.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
```

Comment:
The most important reason is that AFAICT, node-lru does not support EXPIRE operation.

Besides that, LRU cache keeps a fixed amount of items and removes the least-recently-used items when the size limit is hit. IMO, that is very undesirable behaviour for a key-value store implementation.

I guess we could use LRU with `max: Infinity`, but that's not great, as we will be bending the library to our needs that are different from what the library is designed for. In my experience, that usually leads to headaches later in the future, when we hit the limits of how the library can be bent.

> https://github.com/isaacs/node-lru-cache#options
> `max` The maximum size of the cache, checked by applying the length function to all values in the cache. Not setting this is kind of silly, since that's the whole purpose of this lib, but it defaults to Infinity.